src/Presentation/Presentation.csproj

```diff
@@ -9,8 +9,13 @@
    </ItemGroup>
     <ItemGroup>
         <PackageReference Include="GraphQL" Version="4.5.0" />
+        <PackageReference Include="GraphQL.NewtonsoftJson" Version="4.5.0" />
         <PackageReference Include="Microsoft.AspNetCore" Version="1.1.2" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.3" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="5.0.7" />
         <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.1.2" />
     </ItemGroup>
+    <ItemGroup>
+      <Folder Include="GraphQL\Ioc" />
```

Comment:
Should we remove this folder if it`s empty?

Reply:
Done